<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mediaWidget</class>
 <widget class="QWidget" name="mediaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#nameText
{
	font-size:28px;
	color: rgba(200, 200, 200, 1);
}
#infoText
{
	font-size:15px;
	color: rgba(200, 200, 200, 1);
}</string>
  </property>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>-50</y>
     <width>1110</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color:rgba(0,0,0,0);
}</string>
   </property>
   <widget class="QWidget" name="infoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>250</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rightButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="photoLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pictureEntry">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>600</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>pictureEntry</string>
    </property>
   </widget>
   <widget class="QLabel" name="albumCoverLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>280</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leftButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="backButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>BackButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="musicEntry">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>musicEntry</string>
    </property>
   </widget>
   <widget class="QLabel" name="NameText">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>未知歌曲</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoEntry">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>570</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>vedioEntry</string>
    </property>
   </widget>
   <widget class="QLabel" name="playButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="InfoText">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>未知信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="addFileButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>580</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>                 + Choose File</string>
    </property>
   </widget>
   <widget class="QLabel" name="flowEntry">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>flowEntry</string>
    </property>
   </widget>
   <widget class="QLabel" name="flowRenderer">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>flowRenderer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="flowInput">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入网址</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flowStart">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flowStop">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
